feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -210,67 +142,56 @@ fun PreviewScreen(
                     zoomScale = zoomScale,
                     show = zoomScaleShow
                 )
+
+
                 Row(
-                    horizontalArrangement = Arrangement.SpaceAround
+                    modifier =
+                    Modifier
+                        .fillMaxWidth()
+                        .height(IntrinsicSize.Min),
                 ) {
-                    Row {
-                        CaptureButton(
-                            onClick = { viewModel.captureImage() },
-                            onLongPress = { viewModel.startVideoRecording() },
-                            onRelease = { viewModel.stopVideoRecording() },
-                            state = previewUiState.videoRecordingState
-                        )
-                    }
+                    FlipCameraButton(
+                        modifier = Modifier
+                            .weight(1f)
+                            .fillMaxHeight(),
+                        onClick = { viewModel.flipCamera() },
+                        //enable only when phone has front and rear camera
+                        enabledCondition =
+                        previewUiState.currentCameraSettings.back_camera_available
+                                && previewUiState.currentCameraSettings.front_camera_available
+                    )
+                    /*todo: close quick settings on start record/image capture*/
+                    CaptureButton(
+                        modifier = Modifier
+                            .fillMaxHeight(),
+                        onClick = { onImageCapture(viewModel) },
```

Comment:
I put these wrappers here because I wanted to add extra behaviors when performing these actions without bloating this area of the code (not reflected in *this* PR).

i.e. if quick settings screen is open, close it when taking a picture or starting recording. 

or if we wanted to play a sound blip when taking a photo/starting a recording, etc.

Reply:
These side effects should happen in response to the changes from the UiState